.devcontainer/cuda/devcontainer.json

```diff
@@ -40,4 +40,4 @@
   }
   // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
   // "remoteUser": "root"
-}
\ No newline at end of file
+}
```

Comment:
@ethanwee1 Can you please figure out how to remove this gratuitous newline diff?